### PR TITLE
Add performance tracking system proposal

### PR DESCRIPTION
**Pull request by lazyparser**
This is the initial proposal of performance tracking infrastructure, which had been present in the code speed opt SIG monthly meeting, 15-Dec-2020.

### CODE REVIEW

File: projects/infrastructure-for-perf-tracking.adoc [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+* Need commercial toolchains & runtimes run the scripts and send back the performance data to the tracking platform.
+
+
+Table of member organizations and commitments
+
+[cols="<4,<4,1,1,1",options="header,pagewidth",]
+|=============================================================================
+| _Organization_  | _Commitment_ | _Past_ | _2021_ | _2022_
+| PLCT Lab.       | 12 engineer months compiler expertise | X | X |
+| Welcome Inc        | €50k funding |     |     | X
```
Comment (jeremybennett) [inline projects/infrastructure-for-perf-tracking.adoc:161]: I think this line is left over from the template. Unless you really do have Welcome Inc. contributing €50k!

**Review by jeremybennett — APPROVED** [review]
Looks good to me, apart from I think there is a spurious line in the table at the end about supporters (see line comment).  Happy to merge - this can be fixed later, since

**Comment by jeremybennett** [comment]
Looks good to merge. One small change needed, but this is a good framework for review.

File: projects/infrastructure-for-perf-tracking.adoc (hunk 1 above)
Comment (lazyparser) [reply projects/infrastructure-for-perf-tracking.adoc:161]: Oops. I'll submit a new PR to remove this. Thanks :)

**Comment by lazyparser** [comment]
> Looks good to merge. One small change needed, but this is a good framework for review.

Thanks. I'll try to update a new version (PR) before the next meeting.

**Pull request merged**